src/prefs/ApplicationPrefs.cpp

```diff
@@ -82,26 +82,38 @@ void ApplicationPrefs::PopulateOrExchange(ShuttleGui & S)
             *DefaultUpdatesCheckingFlag);
 
          S.AddFixedText(
-            XO("Requires network access."),
+            XO("App update checking requires network access. In order to protect your privacy, Audacity does not store any personal information."),
             false, 470);
 
          S.AddSpace(20);
+         /* i18n-hint: %s will be replaced with "our Privacy Policy" */
+         AccessibleLinksFormatter privacyPolicy(XO("See %s for more info."));
 
+         privacyPolicy.FormatLink(
+            /* i18n-hint: Title of hyperlink to the privacy policy. This is an object of "See". */
+            wxT("%s"), XO("our Privacy Policy"),
+            "https://www.audacityteam.org/about/desktop-privacy-notice/");
```

Comment:
this link can be simplified to https://www.audacityteam.org/desktop-privacy-notice/